client/components/credit-card/docs/example.jsx

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component, Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CreditCard from '..';
+
+class CreditCards extends Component {
+	static displayName = 'CreditCard';
+
+	state = {
+		cards: [
+			{
+				card: '4242',
```

Comment:
`card` seems like a fuzzy term. I _think_ we mean _credit card number_ or maybe _last four digits of a credit card number_ - how would you feel about making this name more specific?

`lastFour` or `cardNumber` or `cardId` or something that makes sense would be possibilities

Reply:
Good call – I was sticking to the format already being used, but no reason to formalize it in the view if it's imperfect – done in https://github.com/Automattic/wp-calypso/pull/24485/commits/22dda8e457d74c6541d16bb7583c706f625065cd